<launch>
  
  <node name='active_pub' pkg='rostopic' type='rostopic' args='pub -r 10 /active std_msgs/Bool "data: true"' />

  <node name='scan_aggregator' pkg='exploration' type='scan_aggregator.py' args="0.0 1.0" output="screen" />

  <group ns='/robot_00/scanner_0'>
    <node name='scan_controller' pkg='exploration' type='scan_controller.py' />
    <node name='scan_planner' pkg='exploration' type='scan_planner.py' />
  </group>

  <group ns='/robot_01/scanner_0'>
    <node name='scan_controller' pkg='exploration' type='scan_controller.py' />
    <node name='scan_planner' pkg='exploration' type='scan_planner.py' />
  </group>

  <rosparam command="load" file="$(find bml_vrep)/param/2r_1i_1m_1c_1s_surface.yml" />

</launch>
